### DISCUSSION TITLE
Small ask

### DISCUSSION THREAD

**Discussion by LsBrns** [actualbudget/releases#26] [Feature ideas]
Hello, new here from YNAB :) Small ask. Would it be possible to add the ability to expand/collapse groups with 1-click?

**Comment by jlongster** [comment]
Can you explain more? You can click the group name one and it should definitely expand and collapse :)

**Reply by Deguello** [reply]
Expand and collapse all, not each category group one at a time.

Ugh. Not sure why I thought this was new.

**Comment by LsBrns** [comment]
Thank you :) I know they can be expended/collapsed individually, but can it
be done to all groups at once?

On Mon, Nov 8, 2021 at 4:14 PM James Long ***@***.***> wrote:

> Can you explain more? You can click the group name one and it should
> definitely expand and collapse :)
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/actualbudget/releases/discussions/26#discussioncomment-1608680>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AWMR6X3VR4JU6YHJS52RN2TULA4T7ANCNFSM5HSRYI2A>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>
-- 
Sent from Gmail Mobile

**Comment by bdoherty** [comment]
Toolkit for YNAB offers this functionality.
![image](https://user-images.githubusercontent.com/335468/142038199-887e7844-626f-48cb-b07b-2e4daaf0df3b.png)

YNAB with Toolkit extension.
![image](https://user-images.githubusercontent.com/335468/142038334-0fc0ebbf-a514-49d4-8fa1-c5fbfc7fee4e.png)

YNAB without Toolkit extension.
![image](https://user-images.githubusercontent.com/335468/142038120-6ce9fcc2-ecb3-4254-8d46-13434b30d27d.png)

Maybe it would like something like this?
![image](https://user-images.githubusercontent.com/335468/142039435-b8766963-f57c-4b5a-a69e-0e46fa1920dc.png)
